services/galley/test/integration/API.hs

```diff
@@ -1326,6 +1326,12 @@ testJoinNonTeamConvGuestLinksDisabled = do
     const (Right (ConversationCoverView convId (Just convName))) === responseJsonEither
     const 200 === statusCode
 
+-- | @SF.Separation @TSFI.RESTfulAPI @S2
```

Comment:
I have already added the mapping for `postJoinCodeConvOk` in this PR https://github.com/wireapp/wire-server/pull/2184

Reply:
PR https://github.com/wireapp/wire-server/pull/2184 was merged into develop. Please merge develop back in your branch now.